usermods/smartnest/readme.md

```diff
@@ -51,11 +56,19 @@ void registerUsermods()
 
 Usermod has no configuration, but it relies on the MQTT configuration.\
 Under Config > Sync Interfaces > MQTT:
-* Enable MQTT check box
-* Set the `Broker` field to: `smartnest.cz`
-* The `Username` and `Password` fields are the login information from the `smartnest.cz` website.
+
+* Enable `MQTT` check box.
+* Set the `Broker` field to: `smartnest.cz` or `3.122.209.170`(both work).
+* Set the `Port` field to: `1883`
+* The `Username` and `Password` fields are the login information from the `smartnest.cz` website (It is located above in the 3 points).
 * `Client ID` field is obtained from the device configuration panel in `smartnest.cz`.
+* `Device Topic` is obtained by entering the ClientID/report , remember to replace ClientId with your real information (Because they can ban your device).
+* `Group Topic` keep the same Group Topic.
+
+Wait `1 minute` after turning it on, as it usually takes a while.  
 
 ## Change log
-2022-09
-* First implementation.
+ 2024-05
```

Comment:
In changelog append the changes instead of removing the old log.

Reply:
Fixed up